package.json

```diff
@@ -18,7 +18,7 @@
     "npm": ">=3.0.0"
   },
   "scripts": {
-    "lint": "aegir-lint",
+    "lint": "eslint -c node_modules/aegir/config/eslintrc.yml src test",
```

Comment:
thanks, but lets try to not actually merge this code, will see if I can figure this out on the aegir side